### PR TITLE
params diff: document --deps option

### PR DESCRIPTION
**Pull request by pared**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

related to #2371

### CODE REVIEW

**Review by jorgeorpinel — APPROVED** [review]

File: content/docs/command-reference/params/diff.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ the <abbr>DVC repository</abbr>, or between a commit and the
 
 ```usage
 usage: dvc params diff [-h] [-q | -v] [--targets [<paths> [<paths> ...]]]
```
Comment (jorgeorpinel) [inline content/docs/command-reference/params/diff.md:12]:
```suggestion
usage: dvc params diff [-h] [-q | -v]
```

**Comment by iesahin** [comment]
Thank you @pared

File: content/docs/command-reference/params/diff.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ the <abbr>DVC repository</abbr>, or between a commit and the
...
-                       [--all] [--show-json] [--show-md] [--no-path]
+                       [--all] [--deps] [--show-json] [--show-md] [--no-path]
                        [a_rev] [b_rev]
```
Comment (jorgeorpinel) [inline content/docs/command-reference/params/diff.md:14]:
```suggestion
                       [--targets [<paths> [<paths> ...]]] [--all]
                       [--deps] [--show-json] [--show-md] [--no-path]
```

**Review by jorgeorpinel — COMMENTED** [review]
Ah wait no, just need to reformat this part (max 27 chars for code blocks). I'll commit these:

**Review by jorgeorpinel — APPROVED** [review]
OK

**Pull request merged**